cmd/kubeadm/app/cmd/init.go

```diff
@@ -224,12 +226,14 @@ func AddInitOtherFlags(flagSet *flag.FlagSet, cfgPath *string, skipPreFlight, sk
 		criSocket, "cri-socket", "/var/run/dockershim.sock",
 		`Specify the CRI socket to connect to.`,
 	)
+
 }
 
 // NewInit validates given arguments and instantiates Init struct with provided information.
 func NewInit(cfgPath string, cfg *kubeadmapi.MasterConfiguration, ignorePreflightErrors sets.String, skipTokenPrint, dryRun bool, criSocket string) (*Init, error) {
 
 	if cfgPath != "" {
+		glog.V(1).Infof("[init] Reading config file from : " + cfgPath)
```

Comment:
So we only have two verbosity levels? 0 (default) and 1, right? might be good to document this somewhere

Reply:
Also, should be: `from:`

Reply:
Good observation for `from:`. I'll take care of that from now on.